source/java/src/org/lucee/extension/cache/eh/util/TypeUtil.java

```diff
@@ -64,31 +76,24 @@ public static Object toJVM(Object value) {
 		// Component
 		// UDF
 		
-		
-		// DateTime
-		if(value instanceof DateTime) {
-			return new Date(((DateTime)value).getTime());
-		}
-		// TimeSpan
-		if(value instanceof TimeSpan) {
-			return ((TimeSpan)value).castToDoubleValue(0);
-		}
 		ClassLoader cl = value.getClass().getClassLoader();
+
 		if(cl!=null && cl!=ClassLoader.getSystemClassLoader()) {
 			try {
-				return "lucee-serialized:"+SerializerUtil.serialize(value);
+				return "lucee-serialized:"+SerializerUtil.serialize((Serializable) value);
+			}
+			catch (Exception e) {
+				print.e("Could not serialize item \"" + value.toString() + "\" in toJVM()");
```

Comment:
print is just for debugging and never should be used in published code. I'm always remove the "extends aprint" from print before i commit, so i see where it is used and i can clean up. you should throw an exception or log it to logger.

Reply:
Good catch. Just left over debug information. I've commented the lines out.